makefile

```diff
@@ -370,7 +371,33 @@ swaggo-build:
 swaggo-render:
 	npx @redocly/cli build-docs $(buildDir)/swagger.json -o $(buildDir)/redoc-static.html
 
-phony += swaggo swaggo-install swaggo-format swaggo-build swaggo-render
+
+# Variables
+# TODO: Update this after DEVPROD-14517 is complete
+OPENAPI_FWS_CONFIG_URL := https://foliage-web-services.cloud-build.staging.corp.mongodb.com/foliage_web_services.json
+OPENAPI_FWS_HARDCODED_CONFIG := thirdparty/clients/foliage_web_services.json
+OPENAPI_FWS_OUTPUT_DIR := thirdparty/clients/fws
+OPENAPI_FWS_CONFIG := packageName=fws,packageVersion=1.0.0,packageTitle=FoliageWebServices,packageDescription="Foliage Web Services",apiTests=false,modelTests=false
+OPENAPI_GENERATOR := bin/openapi-generator-cli.sh
+
+# Main rule for generating the client
+# TODO: Remove the hardcoded config rule once DEVPROD-14517 is complete
+# fws-client: download-fws-config generate-client
+fws-client: generate-fws-client
+
+download-fws-config:
+	@echo "Downloading OpenAPI config..."
+	curl -L -o openapi-config.json $(OPENAPI_FWS_CONFIG_URL)
+
+generate-fws-client:
+	@echo "Generating OpenAPI client..."
+	scripts/setup-openapi-client.sh $(OPENAPI_FWS_HARDCODED_CONFIG) $(OPENAPI_FWS_OUTPUT_DIR) $(OPENAPI_GENERATOR) $(OPENAPI_FWS_CONFIG)
+	@echo "Generating OpenAPI client done."
+	@echo "Swaggo format..."
+	make swaggo-format
```

Comment:
Why does the auto-generating the FWS client need swaggo-format?

Reply:
The [Swaggo tests failed](https://spruce.mongodb.com/task/evergreen_ubuntu2204_verify_swaggo_fmt_patch_d7acb9c1a232c7c845d1fe7c33b04b1a35121692_680bd8c8d5d57b0007148c5a_25_04_25_18_47_37/logs?execution=0) when it was generated due to some misaligned spacing in the generated files